src/zac/demo/mor/tests/factories.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from factory.django import DjangoModelFactory as Dmf
+from zac.accounts.models import User
+from ...models import SiteConfiguration
+
+
+class UserFactory(Dmf):
+
+    class Meta:
+        model = User
+
+    username = 'test'
+    password = 'test1234'
```

Comment:
this does not hash the password, please check other projects or the factory boy documentation on how to set the password